drake/examples/spring_mass/spring_mass_system.h

```diff
@@ -29,7 +29,7 @@ class DRAKESPRINGMASSSYSTEM_EXPORT SpringMassStateVector
   double get_position() const;
 
   /// Sets the position of the mass in meters.
-  void set_position(double v);
+  void set_position(double q);
```

Comment:
note that q was used (in both cases) in the cc file; it was just a typo in the header files:
https://github.com/RobotLocomotion/drake/blob/master/drake/examples/spring_mass/spring_mass_system.cc#L31